### ISSUE TITLE
"Set S_Host =Castle" no longer works...(?)

### ISSUE THREAD

**Issue by Amyrakunejo** [skynettx/raptor#30]
I've tried running a batch file and tried placing the command in the .INI file, and it's not doing anything.

**Comment by skynettx** [comment]
You're probably setting the environment variable incorrectly.
Type this command into cmd in Windows as follows:
```
set S_HOST=CASTLE
```
Under Linux or macOS in the terminal:
```
export S_HOST=CASTLE
```
Then start raptor.exe (Windows) or raptor (Linux or macOS) from the same cmd or terminal session.
If you close the cmd or the terminal you have to set the environment variable again.

You can also create a batch file.
On Windows, for example, create a batch file (.bat) in the working directory of raptor.exe with the following content:
```
set S_HOST=CASTLE
raptor.exe
```
Then you can start Raptor in cheat mode via the .bat without having to set the environment variable every time you close the cmd.

**Comment by Amyrakunejo** [comment]
Nope. It's not working. I know for a fact that with this active, even phase shields take no damage, and unless that's been altered (I doubt it), those take damage still, so I dunno what's going on, but if I had to guess, S_HOST is being overwritten.

**Comment by skynettx** [comment]
I just tested the cheat mode again, it works for me without any problems. 
The environment variable must be set exactly as described above, i.e. pay attention to upper and lower case letters.

**Comment by Amyrakunejo** [comment]
set S_HOST=CASTLE
raptor.exe

This is how I have it set up.
It still, STILL, is not working. Maybe, just maybe, I messed up in the installation? I do have the GoG version.

**Comment by skynettx** [comment]
Here's a video of how it works with release version 0.8.0 under Windows.

https://github.com/skynettx/raptor/assets/83710055/9384df74-3341-45b4-9da4-90e2ccb753d6

When cheat mode is active, the Apogee and Cygnus intros are skipped, the player is in god mode and has all weapons and items (depending on registered and unregistered mode).

**Comment by Amyrakunejo** [comment]
I installed the GoG version, and then installed this on top of that.

Did that mess up anything?

Because, something tells me that it did.

The video shows me that nothing is going as it should; even the intros still play.

Even calling cmd directly and entering the exact commands as shown, this issue persists.

I am going to install this into a different directory to see if that changes anything.

**Comment by Amyrakunejo** [comment]
Reinstalling to a different directory resolved the issue. I am not sure why.

**Comment by SilentMRG** [comment]
> On Windows, for example, create a batch file (.bat) in the working directory of raptor.exe with the following content:
> 
> ```
> set S_HOST=CASTLE
> raptor.exe
> ```

Thanks for that! It worked for me. =) I'm using the files from the ZOOM Platform version: https://www.zoom-platform.com/product/raptor-call-of-the-shadows-1994-classic-edition
